stringtemplate4/src/main/java/org/jdbi/v3/stringtemplate4/StringTemplates.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jdbi.v3.stringtemplate4;
+
+import org.jdbi.v3.core.config.JdbiConfig;
+
+public class StringTemplates implements JdbiConfig<StringTemplates> {
+    private boolean failOnMissingAttribute = false;
+
+    public StringTemplates() {}
+
+    StringTemplates(StringTemplates other) {
+        this.failOnMissingAttribute = other.failOnMissingAttribute;
+    }
+
+    public boolean failOnMissingAttribute() {
+        return failOnMissingAttribute;
```

Comment:
I cringe a bit at the "getter has the same name as field" pattern. `isFailOnMissingAttribute`?

Reply:
Ah, I cringe at unnecessarily modifying property names for their getter which makes all property discovery much harder, since you have to guess what prefix is maybe or maybe not there :/
I will change it.